meeteval/wer/__main__.py

```diff
@@ -99,7 +90,7 @@ def _load_hypothesis(hypothesis: List[Path]):
         raise RuntimeError(hypothesis, filename)
 
 
-def _load_texts(reference_paths: List[str], hypothesis_paths: List[str], regex):
+def _load_texts(reference_paths: List[str], hypothesis_paths: List[str], regex) -> (STM, List[Path], STM, List[Path]):
```

Comment:
`(STM, List[Path], STM, List[Path])` -> `'STM | list[Path]'` (Duplicates?)

For type annotations: I prefer the newest syntax as str:
 - Modern look
 - Works in old python

Reply:
This should mean `Tuple[STM, List[Path], STM, List[Path]]`. This was autofilled by PyCharm (or Copilot?), so I thought it was correct

Reply:
Changed it to `Tuple[STM, List[Path], STM, List[Path]]`.